API.yaml

```diff
@@ -774,6 +774,59 @@ paths:
             application/json:
               schema:
                 type: string
+  /experiments/{expUUID}/outputs/timeGraph/{outputID}/markers:
```

Comment:
I think this should be renamed to _/experiments/{expUUID}/outputs/{outputID}/annotations:_